colossus/src/main/scala/colossus/core/Worker.scala

```diff
@@ -470,11 +396,18 @@ private[colossus] class Worker(config: WorkerConfig) extends Actor with ActorMet
           con.handleConnected()
         } catch {
           case c: java.net.ConnectException => {
+            //TODO consider rolling in connection failure as a disconnect cause
```

Comment:
These 2 catches are pretty much identical, sans the log msg.  Should we combine them?